test/index.js

```diff
@@ -0,0 +1,176 @@
+const expect = require('expect');
+const {createRobot} = require('probot');
+const plugin = require('..');
+const commentEvent = require('./fixtures/issue_comment.created');
+
+describe('snooze', () => {
+  let robot;
+  let github;
+
+  beforeEach(() => {
+    robot = createRobot();
+
+    // Load the plugin
+    plugin(robot);
```

Comment:
aAnd 2 days of investigation I finally understand enough of the code to understand what's happening here. And yes, @bkeepers , you were spot on days ago. That was a big piece of this success.